src/test/K6/src/tests/platform/authorization/delegations/inheritance.js

```diff
@@ -59,20 +81,20 @@ export default function (data) {
     coveredBy: 'urn:altinn:userid',
     resource: ['urn:altinn:app', 'urn:altinn:org'],
   };
-  res = delegation.getRules(altinnToken, policyMatchKeys, 123, 999, resources, null, [456]);
+  res = delegation.getRules(altinnToken, policyMatchKeys, partyId1, 999, resources, null, [userId2]);
   success = check(res, {
     'Inherited Via KeyRole - status is 200': (r) => r.status === 200,
     'Inherited Via KeyRole - rule id matches': (r) => r.json('0.ruleId') === ruleId,
     'Inherited Via KeyRole - createdSuccessfully is false': (r) => r.json('0.createdSuccessfully') === false,
-    'Inherited Via KeyRole - offeredByPartyId matches': (r) => r.json('0.offeredByPartyId') === 123,
+    'Inherited Via KeyRole - offeredByPartyId matches': (r) => r.json('0.offeredByPartyId') === partyId1,
     'Inherited Via KeyRole - coveredBy is partyid': (r) => r.json('0.coveredBy.0.id') === 'urn:altinn:partyid',
-    'Inherited Via KeyRole - coveredBy matches': (r) => r.json('0.coveredBy.0.value') === '456',
+    'Inherited Via KeyRole - coveredBy matches': (r) => r.json('0.coveredBy.0.value') === userId2.toString(),
```

Comment:
Her sjekkes det om coveredBy 'urn:altinn:partyid', men man sjekker at value er userId2

Reply:
Ser det også er opprettet mye med CoveredByPartyId: 20002886 i AT22 nå som feiler

Reply:
I replaced the userid with partyid for the first half of the tests